core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -786,7 +786,7 @@ class HttpGoogleServicesDAO(
       "projectOwnersGroup" -> ownerGroupEmail.value.toJson,
       "projectViewersGroup" -> computeUserGroupEmail.value.toJson,
       "requesterPaysRole" -> requesterPaysRole.toJson,
-      "highSecurityNetwork" -> false.toJson,
+      "highSecurityNetwork" -> true.toJson, //turned on for all new projects
```

Comment:
Have you taken into account what you _mean_ by "true" in this context?  I mean, what is truth?  Is it not just a construct of human blah blah blah, whatever

Reply:
yes